fdbserver/storageserver.actor.cpp

```diff
@@ -3073,73 +3076,66 @@ ACTOR Future<Void> updateStorage(StorageServer* data) {
 		state Promise<Void> durableInProgress;
 		data->durableInProgress = durableInProgress.getFuture();
```

Comment:
This is also set unconditionally in the loop below.

Reply:
```suggestion
```